packages/sdk/src/oracle/tests/optimisticV2.e2e.ts

```diff
@@ -0,0 +1,94 @@
+import dotenv from "dotenv";
```

Comment:
this is a new e2e test which just makes sure things build and we can get a request on polygon